core/src/main/java/hudson/model/UnprotectedRootAction.java

```diff
@@ -32,4 +32,26 @@
  * @since 1.398
  */
 public interface UnprotectedRootAction extends RootAction, ExtensionPoint {
```

Comment:
The override should be in `RootAction`. `UnprotectedRootAction` adds no special behavior w.r.t. the URL.